client/app/components/category_list.vue

```diff
@@ -3,19 +3,42 @@
 
     block content
         h1 {{ 'my_accounts category title' | t }}
-        article {{ 'my_accounts coming_soon' | t }}
-
-        ul
-            li
-                button(v-on:click="openDialog") open dialog
+        div.konnectors-list
+            konnector-item(
+                v-for="konnector in konnectorsList",
+                v-if="!konnector.accounts.length",
+                :title="konnector.name",
+                :subtitle="(konnector.category + ' category') | t",
+                :backgroundCSS="konnector.color.css",
+                :enableDefaultIcon="true",
+                :iconName="konnector.slug",
+                link="#")
 </template>
 
 <script>
+    import konnectorItemComponent from './list_item'
+
     export default {
-      methods: {
-          openDialog () {
-              this.$emit('open-dialog', 'dialog-1')
-          }
-      }
+        data: function () {
+            return {
+                konnectorsList: window.initKonnectors
```

Comment:
We shouldn't use the list from the `window` directly inside the component and externalise it or we shouldn't use it at all? For the first case I can put it in an helper for better reusability.